css/modules.css

```diff
@@ -0,0 +1,55 @@
+div {
```

Comment:
I would think most (or even all) of this css belongs in the `layout` file: it all describes the basic way your page is displayed.